tool/cargo/sync_properties_aspect.bzl

```diff
@@ -171,21 +170,21 @@ def _build_sync_info_file(target, ctx, source_files, crate_info):
     )
     return file
 
-def _rust_ide_sync_info_aspect_impl(target, ctx):
+def _rust_cargo_sync_properties_aspect_impl(target, ctx):
     if ctx.rule.kind not in _TARGET_TYPES.keys():
         return struct(kind = ctx.rule.kind)
 
     sources = [f for src in getattr(ctx.rule.attr, "srcs", []) for f in src.files.to_list()]
     crate_info = _crate_info(ctx, target)
-    sync_info_file = _build_sync_info_file(target, ctx, sources, crate_info)
+    sync_info_file = _build_cargo_sync_properties_file(target, ctx, sources, crate_info)
```

Comment:
Right. I'd (incorrectly!) said previously that the term "Info" carries no meaning in Bazel. This isn't quite accurate. [Providers](https://bazel.build/rules/lib/Provider) are conventionally named `.*Info`. [Structs](https://bazel.build/rules/lib/struct) are slightly less strongly-typed but often more lightweight and can be used in place of providers - as we do here, returning a `struct` from the `...aspect_impl` method as opposed to a provider.

That being said, this method was always a little awkward, because the struct was called "sync info", and the output file was called "sync info", and guess what, they contained different content. So now we can fix that situation.

But - now that we've resolved that ambiguity, let's make sure we apply it consistently! This variable, for instance, `sync_info_file`, should be renamed to `cargo_sync_properties_file`. (And maybe we should use `props` throughout for brevity?)

Reply:
I just missed this variable seems like, have updating the naming here :)